src/js/health-records/containers/Main.jsx

```diff
@@ -173,12 +178,20 @@ export class Main extends React.Component {
     const customDateRangeError =
       this.state.invalidStartDateFormat ||
       this.state.invalidEndDateFormat;
+    const missingDateError =
+      this.state.missingStartDate ||
+      this.state.missingEndDate;
 
     const customDateOptionClass = classNames({
       'custom-date-option': true,
       'date-range-error': customDateRangeError
```

Comment:
This should probably include both error types. `'date-range-error': invalidFormatError || missingDateError`.